code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -685,3 +685,14 @@ new /datum/stack_recipe("paper frame door", /obj/structure/mineral_door/paperfra
 	merge_type = /obj/item/stack/sheet/meat
 	material_type = /datum/material/meat
 	material_modifier = 1 //None of that wussy stuff
+
+/obj/item/stack/sheet/obamium
+	name = "obamium sheet"
+	desc = "Finally...Obamium"
```

Comment:
add a space after this ellipsis.

also consider adding a period to this description.

Reply:
why dont u use code suggestions for things like this :^(

Reply:
```suggestion
	desc = "Finally... Obamium..."
```